src/query/storages/fuse/src/fuse_table.rs

```diff
@@ -286,19 +291,28 @@ impl FuseTable {
         }
     }
 
+    #[minitrace::trace]
+    #[async_backtrace::framed]
```

Comment:
Maybe we can use `SnapshotsIO::read_snapshot` directly?
https://github.com/datafuselabs/databend/blob/60004ed525cb2e122e0b5b07810c853ec5b0b617/src/query/storages/fuse/src/io/snapshots.rs#L78-L92